simcore/simulation/library/anchor.cpp

```diff
@@ -195,6 +197,20 @@ void Anchor::UpdateAnchorPositionToBond() {
   }
   UpdatePeriodic();
 }
+/*Creates Vector that has different binding rates for parallel and anti-parallel bonds*/
+std::vector<double> Anchor::CreateOrientationArray(int n_neighbors){
+  double const *const orientation = bond_->GetOrientation();
+   std::vector<double> OrientationArray(n_neighbors, k_on_d_);     
```

Comment:
Variable names should be in the form lower_case_with_underscores. Class names and methods are CamelCase.